xbmc/utils/CryptThreading.cpp

```diff
@@ -55,7 +69,9 @@ CryptThreadingInitializer::~CryptThreadingInitializer()
 {
 #if KODI_OPENSSL_NEEDS_LOCK_CALLBACK
   CSingleLock l(m_locksLock);
+#if ! KODI_OPENSSL_USE_THREADID
```

Comment:
```suggestion
# if !KODI_OPENSSL_USE_THREADID
```

Reply:
please use `# if !defined(KODI_OPENSSL_USE_THREADID)` - I've seen some compilers choke on this

Reply:
ok, nvm